resume <- tags$div(class="col-sm-8",
                   HTML(


'<p dir="ltr">
  <h3>Andrea McEwan | (647) 537-0300 | andrea.h.mcewan@gmail.com</h3>
    <hr/>
      </p>
      <p dir="ltr">
       <b> SUMMARY</b>
      </p>
        <hr/>
        <ul>
        <li dir="ltr">
          <p dir="ltr">
            Highly proficient in R programming; experience creating R-Shiny
          tools for data analysis
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Experience running analysis pipelines for genetic screens used to
          identify targets in cancer research
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Motivated self-starter that takes initiative and can work with
          minimal supervision
          </p>
            </li>
            </ul>
            <br/>
            <p dir="ltr">
          <b>  EDUCATION </b>
          </p>
            <hr/>
            <br/>
            <p dir="ltr">
            Certificate in Data Analytics, Big Data, and Predictive Analytics | Ryerson University
          </p>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Machine Learning Techniques
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Programming in R and big data programming tools
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Statistics
          </p>
            </li>
            </ul>
            <p dir="ltr">
            M.Sc Molecular and Developmental Biology | University of British Columbia
          </p>
            <p dir="ltr">
            B.Sc. Biochemistry | University of Ottawa
          </p>
            <br/>
            <p dir="ltr">
            <b>EMPLOYMENT</b>
          </p>
            <hr/>
            <br/>
            <p dir="ltr">
            <b>2018-Current: Verification Specialist with Analytics (SCIEX) </b>
          </p>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Co-created an R-Shiny application to mine text including internal
          documents and public repositories
          </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Created tool to identify trends in mass spectrometry service
          requests
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Presented the results to the leadership team, including the VP
          of Global Marketing and Strategy and the VP of Software
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Documented algorithms and recapitulated calculations used in mass
          spectrometry analysis software
          </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Researched and documented algorithms used in analysis.
          Disseminated information to Verification team and the Director
          of Applications to aid in customer inquiries
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Produced Excel files that demonstrated how the calculations are
          performed
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Gave software demos to internal and third party team members
          </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Demonstrated SCIEX software to third party vendors
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Created documents and instructions to answer customer inquiries
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Routinely communicated novel software behaviour to Verification
          team
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <br/>
            <p dir="ltr">
            <b>2015-2018: Laboratory Technician III (Lunenfeld Tanenbaum Research
                                                  Institute) Mount Sinai, Toronto, Ontario </b>
          </p>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Created R-based web application used to analyze large datasets
          including:
            </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            An application that takes data from cells stained with EdU and
          DAPI, cleans data and splits it into collections based on the
          cell cycle stage. This application saves the time taken to
          manually copy and paste dozens of Excel files and allows a
          quick way of checking the quality of the experiment. 
          (<a href="https://durocherlab.shinyapps.io/splitcell/">https://durocherlab.shinyapps.io/splitcell/</a>)
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            An application that quickly plots fold change values of sgRNAs
          in wildtype and mutant samples from CRISPR-Cas9 screens and
          allows for quick evaluation of guide performance.
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Analyzed CRISPR-Cas9 screen data including generating read-count
          values from fastq files, creating quality control reports and
          ranking genes for their potential in cancer therapeutics
          </p>
            </li>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Created visualizations of CRISPR-Cas9 results screen using R and
          Cytoscape which were used in presentations and publications
          </p>
            </li>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Predicted genetic interactions using supervised and unsupervised
          learning algorithms in cancer cell lines:
            </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Used PCA to rank strongest context-specific hit in CRISPR-Cas9
          screens
          </p>
            </li>
            <li dir="ltr">
            <p dir="ltr">
            Used hierarchical clustering to determine functionally similar
          genes including HELQ and C17orf53
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Debugged publicly available academic software used for reagent
          design:
            </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Fixed bugs in sgTiler, a tool used to design tiling sgRNA
          libraries for CRISPR-Cas9 screens (<a href="https://github.com/HansenHeLab/sgTiler">https://github.com/HansenHeLab/sgTiler</a>)
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Built an online database of laboratory reagents using
          MySQL/PHP/Bootstrap which improved access to reagent information
          </p>
            </li>
            </ul>
            <br/>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Collaborated with the Network Biology Collaborative Centre
          </p>
            </li>
            <ul>
            <li dir="ltr">
            <p dir="ltr">
            Used Python to create an automated email-alert system that
          informed the High-Content Screening Manager at the Network
          Biology Collaborative Centre when a high-throughput microscope
          had failed to upload data to a central repository
          </p>
            </li>
            </ul>
            </ul>
            <br/>
            <br/>
            <p dir="ltr">
          <b>  Selected publications </b>
          </p>
            <br/>
            <ol>
            <li dir="ltr">
            <p dir="ltr">
            Zimmerman M, Murina O, Reijns MAM, Agathanggelou A, Challis R,
          Tarnauskaite, Muir M, Fluteau A, Aregger M, McEwan A, Yuan W,
          Clarke M, Lambros MB, Paneesha S, Moss P, Chandrashekhar M, Angers
          S, Moffat J, Brunton VG, Hart T, de Bono J, Stankovic T, Jackson
          AP, Durocher D. (2018)<b> CRISPR screens identify genomic
          ribonucleotides as a source of PARP-trapping lesions</b>. Nature volume
          559, pages 285-289
          </p>
            </li>
            </ol>
            <br/>
            <ol start="2">
            <li dir="ltr">
            <p dir="ltr">
            Sylvie M. Noordermeer, Salome Adam, Dheva Setiaputra, Marco
          Barazas, Stephen J. Pettitt, Alexanda K. Ling, Michele Olivieri,
          Alejandro √Ålvarez-Quilon, Nathalie Moatti, Michal Zimmermann,
          Stefano Annunziato, Dragomir B. Krastev, Feifei Song, Inger
          Brandsma, Jessica Frankum, Rachel Brough, Alana Sherker, Sebastien
          Landry, Rachel K. Szilard, Meagan M. Munro, Andrea McEwan, Theo
          Goullet de Rugy, Zhen-Yuan Lin, Traver Hart, Jason Moffat,
          Anne-Claude Gingras, Alberto Martin, Haico van Attikum, Jos
          Jonkers, Christopher J. Lord, Sven Rottenberg and Daniel Durocher
          (2018). <b>The Shieldin complex mediates 53BP1-dependent DNA repair</b>.
          Nature volume 560, pages 117-121
          </p>
            </li>
            </ol>
            <br/>
            <ol start="3">
            <li dir="ltr">
            <p dir="ltr">
            Wilson, M., Benlekbir, S., Fradet-Turcotte, A., Sherker, A.,
          Julien, J-P., McEwan, A., Noordermeer, S., Sicheri, F., Rubinstein,
          J. &amp; Durocher, D. (2016) <b>The structural basis of modified
          nucleosome recognition by 53BP1</b>. Nature volume 536; 536:100-103
          </p>
            </li>
            </ol>')
)


            